### PR TITLE
Add: cert-manager and letsencrypt into resources certificates

### PR DESCRIPTION
**Pull request by nlamirault**
Signed-off-by: Nicolas Lamirault <nicolas.lamirault@gmail.com>

### CODE REVIEW

File: diagrams/onprem/certificates.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+from . import _OnPrem
+
+
+class _Certificates(_OnPrem):
+    _type = "certificates"
+    _icon_dir = "resources/onprem/certificates"
+
+
+class Certmanager(_Certificates):
```
Comment (mingrammer) [inline diagrams/onprem/certificates.py:11]: I request some changes similar to [this request change](https://github.com/mingrammer/diagrams/pull/260#discussion_r487851767).

It should be `class CertManager`.

File: diagrams/onprem/certificates.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    _icon = "certmanager.png"
+
+
+class Letsencrypt(_Certificates):
```
Comment (mingrammer) [inline diagrams/onprem/certificates.py:15]: I request some changes similar to [this request change](https://github.com/mingrammer/diagrams/pull/260#discussion_r487851767).

It should be `class LetsEncrypt`.

**Review by mingrammer — CHANGES_REQUESTED** [review]

File: diagrams/onprem/certificates.py (hunk 2 above)
Comment (nlamirault) [reply diagrams/onprem/certificates.py]: Done.

**Review by mingrammer — APPROVED** [review]
LGTM!

**Pull request merged**